src/Common/Compat.targets

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
I'm not sure yet where do we want to put this targets file, so I'm setting it here for now. I'm happy to change it to buildtools if that's what we want.